content/pages/volunteers.md

```diff
@@ -0,0 +1,96 @@
+Title: Volunteer Opportunities
+Category: pages
+Summary: Volunteer opportunities available with WiCS undergraduate.
+
+
+As we strive to improve the quality of our events, every contribution, large or
+small counts. Volunteering with our committee is a great way to get involved
+with us on a smaller scale. While being a committee member requires you to
+dedicate at least 8 hours per week to us, becoming a core volunteer is an
+easier, lower commitment way to be involved in our activities, help us run
+events and workshops, and get to know the committee members.  Volunteering is
+also a good way to figure out if you'd like to commit more time to WiCS and
+eventually apply to be a committee member.
+
+We have a range of volunteer positions, each of which will help you to learn
+about an aspect of our committee. By volunteering to be a **photographer**, you
+can learn about how we run events and help with publicizing them. By
+volunteering to be a **technical mentor** for our events, you can help teach
+technical concepts to others. By becoming a **wics-sys volunteer**, you can
+learn systems administration and static website development from our systems
+committee.
+
+## How volunteering works
+
+You can sign up for most positions by filling out
+[this](http://goo.gl/forms/NsOXea6Yjf5t02Oj1) form. The only exception is
```

Comment:
also I'm worried that because the form we're linking to is term specific, we need to make changes to the site every term

what if the form is general, but we filter responses based on the date they fill it out? then we don't have to make a new form and change the website every term